commands/purge.js

```diff
@@ -12,6 +12,10 @@ module.exports = {
     async execute(interaction) {
         const deleteCount = interaction.options.get('num').value;
 
+        if (!interaction.member.permissions.has('MANAGE_MESSAGES')) {
+            return interaction.reply("I can't purge these messages.");
```

Comment:
Please add a more descriptive error message so the user knows about the permission issue.